src/test/java/step2/StringCalculatorTest.java

```diff
@@ -0,0 +1,46 @@
+package step2;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static step2.StringCalculator.*;
+
+public class StringCalculatorTest {
+
+    @Test
+    void 빈문자열_null() {
+        assertThat(splitAndSum(null)).isEqualTo(0);
+        assertThat(splitAndSum("")).isEqualTo(0);
+    }
+
+    @Test
+    void 숫자_하나() {
+        assertThat(splitAndSum("1")).isEqualTo(1);
+    }
+
+    @Test
+    void 컴마구분자() {
+        assertThat(splitAndSum("1,2")).isEqualTo(3);
+    }
+
+    @Test
+    void 쉼표_콜론구분자() {
+        assertThat(splitAndSum("1,2:3")).isEqualTo(6);
+    }
+
+    @Test
+    void custom_구분자() {
+        assertThat(splitAndSum("//;\n1;2;3")).isEqualTo(6);
+    }
+
+    @Test
+    void 음수() {
+        assertThatThrownBy(() -> splitAndSum("-1,2:3")).isInstanceOf(RuntimeException.class);
```

Comment:
여러 경우의 RuntimeException이 발생 할 수 있어서, 에러메세지도 같이 확인 해주면 좋을 것 같아요 :) 

```suggestion
        assertThatThrownBy(() -> splitAndSum("-1,2:3"))
            .isInstanceOf(RuntimeException.class)
            .hasMessage("음수는 입력할 수 없습니다.");

```

Reply:
감사합니다 ~! 앞으로 더 신경써보도록 하겠습니다